### DISCUSSION TITLE
Inkscape directory not found

### DISCUSSION THREAD

**Discussion by underscorelsa** [sleepymalc/VSCode-LaTeX-Inkscape#27] [Q&A]
Hi! Inkscape watch works great, but when I just try running the Inkscape create and edit commands, there's a message on the runCommand terminal that reads:
FileNotFoundError: [Errno 2] No such file or directory: 'inkscape'

Could anyone help me fix this, please?

**Comment by sleepymalc** [comment]
Sorry for the late answer. Can you check whether you can run Inkscape in the terminal? I.e., check what `which inkscape` give you. It should give you the path of that executable.
